Failing build

2026-05-28T22:42:27.5207477Z ##[group]Run npm run build
2026-05-28T22:42:27.5207793Z [36;1mnpm run build[0m
2026-05-28T22:42:27.5233686Z shell: /usr/bin/bash -e {0}
2026-05-28T22:42:27.5233939Z env:
2026-05-28T22:42:27.5234157Z   VITE_BASE_PATH: /incometrack/
2026-05-28T22:42:27.5234409Z ##[endgroup]
2026-05-28T22:42:27.6313007Z 
2026-05-28T22:42:27.6313711Z > incometrack@0.0.0 build
2026-05-28T22:42:27.6314154Z > tsc -b && vite build
2026-05-28T22:42:27.6314350Z 
2026-05-28T22:42:32.9996576Z ##[error]src/components/scenarios/LifetimeProjectionChart.tsx(90,21): error TS2339: Property 'activePayload' does not exist on type 'MouseHandlerDataParam'.
2026-05-28T22:42:33.0009970Z ##[error]src/components/scenarios/LifetimeProjectionChart.tsx(91,34): error TS2339: Property 'activePayload' does not exist on type 'MouseHandlerDataParam'.
2026-05-28T22:42:33.0013814Z ##[error]src/components/scenarios/LifetimeProjectionChart.tsx(95,21): error TS2339: Property 'activePayload' does not exist on type 'MouseHandlerDataParam'.
2026-05-28T22:42:33.0018022Z ##[error]src/components/scenarios/LifetimeProjectionChart.tsx(96,34): error TS2339: Property 'activePayload' does not exist on type 'MouseHandlerDataParam'.
2026-05-28T22:42:33.0021464Z ##[error]src/pages/CashflowProjectionPage.tsx(1,1): error TS6133: 'React' is declared but its value is never read.
2026-05-28T22:42:33.3362569Z ##[error]Process completed with exit code 2.

--- a/src/components/scenarios/LifetimeProjectionChart.tsx
+++ b/src/components/scenarios/LifetimeProjectionChart.tsx
@@ -86,12 +86,12 @@ const LifetimeProjectionChart: React.FC = () => {
           <AreaChart
             data={data}
             margin={{ top: 10, right: 10, left: -20, bottom: 20 }}
-            onMouseMove={(e) => {
+            onMouseMove={(e: any) => {
               if (e.activePayload) {
                 setActivePoint(e.activePayload[0].payload);
               }
             }}
-            onTouchMove={(e) => {
+            onTouchMove={(e: any) => {
               if (e.activePayload) {
                 setActivePoint(e.activePayload[0].payload);
               }

--- a/src/pages/CashflowProjectionPage.tsx
+++ b/src/pages/CashflowProjectionPage.tsx
@@ -1,4 +1,3 @@
-import React from 'react';
 import { AppLayout } from '../components/layout/AppLayout';
 import { Header } from '../components/layout/Header';
 import { Icon } from '../components/ui/Icon';

